k8s/aat/common/reform-scan/notification-service.yaml

```diff
@@ -21,6 +28,30 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/reform-scan/notification-service:prod-8cdadccd
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        keyVaults:
+          s2s:
+            resourceGroup: s2s
+            secrets:
+              microservicekey-bulk-scan-processor-tests: S2S_SECRET
+          reform-scan:
+            resourceGroup: reform-scan
+            secrets:
+              notifications-queue-listen-connection-string: NOTIFICATION_QUEUE_CONN_STRING_READ
```

Comment:
staging?

Reply:
staging is for jenkins pipelines as it deploys to staging pods. flux deploys to "real" stuff so - not staging. unless i'm getting it wrong